app/Providers/ViewComposerServiceProvider.php

```diff
@@ -31,6 +31,7 @@ public function boot()
     public function register()
     {
         $this->app->view->composer('*', 'CachetHQ\Cachet\Composers\LoggedUserComposer');
+        $this->app->view->composer('*', 'CachetHQ\Cachet\Composers\ThemeComposer');
```

Comment:
Probably a good idea, for some reason I was thinking of more pages than what we have...